policy/options.go

```diff
@@ -107,3 +115,17 @@ func (o *options) SetDefaultGaugeAggregationTypes(aggTypes AggregationTypes) Opt
 func (o *options) DefaultGaugeAggregationTypes() AggregationTypes {
 	return o.defaultGaugeAggregationTypes
 }
+
+func (o *options) IsContainedInDefaultAggregationTypes(at AggregationType, mt metric.Type) bool {
+	var aggTypes AggregationTypes
+	switch mt {
+	case metric.CounterType:
+		aggTypes = o.DefaultCounterAggregationTypes()
+	case metric.GaugeType:
+		aggTypes = o.DefaultGaugeAggregationTypes()
+	case metric.TimerType:
+		aggTypes = o.DefaultTimerAggregationTypes()
```

Comment:
Add a `default: return false` to be safe? Otherwise it panics.

Reply:
Nope, it's just an empty aggTypes, won't panic